src/test/java/ladder/domain/LadderGameTest.java

```diff
@@ -0,0 +1,36 @@
+package ladder.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.BooleanSupplier;
+import ladder.domain.linegenerator.LinePatternGenerator;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class LadderGameTest {
+    private List<PlayerName> playerNames;
+    private Ladder ladder;
+
+    @Test
+    @DisplayName("게임 결과를 확인한다")
+    void playGameTest() {
+        List<PlayerName> playerNames = new ArrayList<>();
+        playerNames.add(new PlayerName("hogee"));
+        playerNames.add(new PlayerName("jazz"));
+        playerNames.add(new PlayerName("pola"));
```

Comment:
```suggestion
        List<PlayerName> playerNames = List.of(
            new PlayerName("pobi"),
            new PlayerName("crong"),
            new PlayerName("honux")
        );
```
자바에서 제공하는 편리한 방식을 적극 활용해보죠!